inference-engine/tests/ngraph_functions/src/utils/ngraph_helpers.cpp

```diff
@@ -169,8 +171,8 @@ std::vector<std::vector<std::uint8_t>> getConstData(const std::shared_ptr<Functi
         const auto dataSize = shape_size(parrentNode->get_shape()) * parrentNode->get_element_type().size();
         outputs[i].resize(dataSize);
         std::copy(data, data + dataSize, outputs[i].data());
-        if (convertType != element::Type_t::undefined)
-            outputs[i] = convertOutputPrecision(outputs[i], parrentNode->get_element_type(), convertType, shape_size(parrentNode->get_shape()));
+        if (!convertType.empty() && convertType[i] != element::Type_t::undefined)
```

Comment:
if `convertType.empty()` is `true`, we won't enter cycle.

Reply:
```convertType``` is empty in case if we don`t specify output precision and will be compare results as is